File: URI/concluidos/Javascript/1035.js
var input = require('fs').readFileSync('./dev/stdin/arquivo.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (line) => line.split(" ").map(a => +a);
const [a, b, c, d] = pegarValores(lines.shift());

var somaCD = c+d;
var somaAB = a+b;

if ((b > c) && (d > a) && (somaCD > somaAB) && (c > 0) && (d > 0) && (a%2 === 0)){
    console.log('Valores aceitos');
}else{
    console.log('Valores nao aceitos');
}

File: URI/concluidos/Python/1035.py
linha = input()

[a,b,c,d] = linha.split(" ")
a = int(a)
b = int(b)
c = int(c)
d = int(d)

apar = a % 2 == 0
if b > c and d > a and c > 0 and d > 0 and c + d > a + b and apar:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')